autoblocks/_impl/testing/models.py

```diff
@@ -1,10 +1,29 @@
 import abc
 import dataclasses
 import functools
+import uuid
 from typing import Any
+from typing import Dict
 from typing import Optional
 
 
+@dataclasses.dataclass()
+class TracerEvent:
+    message: str
+    trace_id: str
+    timestamp: str
+    properties: dict
+
+    @classmethod
+    def to_json(cls, event: Dict[str, any]):
```

Comment:
This type is not correct, event is an instance of TracerEvent. Also this doesn't need to be a class method, it should just act on the instance (self)